### PR TITLE
ROX-25524: Add enable/disable policy option on create

### PR DESCRIPTION
**Pull request by bradr5**
### Description

Users should be able to enable or disable a policy during creation.

Additionally, combine the "Enforcement" and "Notifiers" steps into a new "Actions" step, where the enable/disable option will be set.

### User-facing documentation

- [ ] CHANGELOG is updated **OR** update is not needed
- [ ] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request) is created and is linked above **OR** is not needed

### Testing and quality

- [ ] the change is production ready: the change is GA or otherwise the functionality is gated by a feature flag
- [ ] CI results are inspected

#### Automated testing

- [ ] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [ ] modified existing tests

#### How I validated my change

Actions Step:
Screenshot 2024-10-07 at 12 14 54 PM

Result:
Screenshot 2024-09-24 at 3 15 30 PM

### CODE REVIEW

File: ui/apps/platform/src/Containers/Policies/Wizard/Step1/PolicyMetadataFormSection.tsx [hunk 1]
```diff
@@ -23,6 +23,14 @@ function PolicyMetadataFormSection(): ReactElement {
 
     return (
         <Form>
+            <Checkbox
+                label="Disable policy"
+                isChecked={values.disabled}
+                onChange={handleChange}
+                description="When checked, this policy will be disabled, stopping enforcement and alerts. By default, the policy is enabled to ensure continuous monitoring and protection."
```
Comment (pedrottimark) [inline ui/apps/platform/src/Containers/Policies/Wizard/Step1/PolicyMetadataFormSection.tsx:30]: An observation, not necessarily a change request.

The prose has a **chiasm**, meaning **crossing over**, from Greek chi which has X shape.
* enforcement / monitoring
* alerts / protection

The correspondence seems more like:
* alerts / monitoring
* enforcement / protection

Comment (bradr5) [reply ui/apps/platform/src/Containers/Policies/Wizard/Step1/PolicyMetadataFormSection.tsx:30]: I should've noted this was made up on the fly and @mansursyed will be providing updated text. Good comment nonetheless.

File: ui/apps/platform/cypress/integration/policies/policiesTable.test.js [hunk 2]
```diff
@@ -286,7 +286,7 @@ describe('Policies table', () => {
         cy.intercept('POST', `${api.policies.policies}?enableStrictValidation=true`).as(
             'POST_policies'
         );
-        cy.get(selectors.wizardBtns.step7).click();
+        cy.get(selectors.wizardBtns.step6).click();
```
Comment (pedrottimark) [inline ui/apps/platform/cypress/integration/policies/policiesTable.test.js:289]: An observation for whenever we next invest in integration tests.

Although there would be some code change any way we organize it:
1. Move and rename policies.js from helpers to policies/Policies/helpers.js
2. Encapsulate selector in something like `goToStep6` helper function.
3. Encapsulate the next 2 lines in something like `savePolicy` helper.

File: ui/apps/platform/src/Containers/Policies/Wizard/Step5/NotifiersForm.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                        />
+                        <Th>Notifier</Th>
+                        <Th>Type</Th>
+                    </Tr>
+                </Thead>
+                <TbodyUnified
+                    tableState={tableState}
+                    colSpan={3}
+                    errorProps={{
+                        title: 'There was an error loading the collections',
```
Comment (pedrottimark) [inline ui/apps/platform/src/Containers/Policies/Wizard/Step5/NotifiersForm.tsx:64]: This looks like copy-paste from the original code?

**Review by pedrottimark — APPROVED** [review]
Clear changes. Possible correction to original text.

**Pull request merged**